muikku-school-data-pyramus/src/main/java/fi/otavanopisto/muikku/plugins/schooldatapyramus/PyramusUpdater.java

```diff
@@ -628,6 +628,15 @@ public boolean updateCourseStudent(Long courseStudentId, Long courseId, Long stu
     
     return false;
   }
+  
+  public void removeCourseStudent(Long courseStudentId, Long courseId, Long studentId) {
+    String identifier = identifierMapper.getWorkspaceStudentIdentifier(courseStudentId);
+    SchoolDataIdentifier workspaceUserIdentifier = new SchoolDataIdentifier(identifier, SchoolDataPyramusPluginDescriptor.SCHOOL_DATA_SOURCE);
+    WorkspaceUserEntity workspaceUserEntity = workspaceUserEntityController.findWorkspaceUserEntityByWorkspaceUserIdentifierIncludeArchived(workspaceUserIdentifier);
+    if (workspaceUserEntity != null) {
+      workspaceUserEntityController.deleteWorkspaceUserEntity(workspaceUserEntity);
```

Comment:
It's deleting WorkspaceUserEntity, I was just wondering if anything has dependencies (like answers on tasks etc) on it, can it cause some sort of headache?